apps/server/src/modules/tool/external-tool/uc/external-tool.uc.spec.ts

```diff
@@ -382,6 +414,14 @@ describe(ExternalToolUc.name, () => {
 				};
 			};
 
+			it('should call encryption service', async () => {
```

Comment:
`it('should call encryption service',` but you `expect(encryptionService.encrypt).not.toHaveBeenCalled()`